app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -77,6 +77,7 @@ const BACKGROUND_METRICS = [
 const NOTIFICATION_TYPES = {
   invalidSamples: "invalid samples",
```

Comment:
nit: if using underscores as the new convention, would you mind refactoring the other notification types to be the same format?